exam -01/section-2-3.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```

Comment:
Section 2: 10/15 the wrong selector `data-component="add"` will cause add task not working which lead to the whole app not working properly

Section 3: 0/15 - You have not done any CSS